go/kbfs/libkbfs/folder_block_manager.go

```diff
@@ -1175,13 +1112,12 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 		}
 	}()
 
-	mostRecentOldEnoughRev, lastGCRev, err :=
-		fbm.getMostRecentOldEnoughAndGCRevisions(ctx, head.ReadOnly())
+	lastGCRev, err := fbm.getMostRecentGCRevision(ctx, head.ReadOnly())
 	if err != nil {
 		return err
 	}
-	if mostRecentOldEnoughRev == kbfsmd.RevisionUninitialized ||
-		mostRecentOldEnoughRev <= lastGCRev {
+	if head.Revision() <= lastGCRev {
+		fbm.log.CDebugf(ctx, "QR NOT NEEDED %d", lastGCRev)
```

Comment:
This probably doesn't need to be all caps.

Reply:
Oops, I meant to take this debugging line out entirely.  Will do, thanks!